src/gui/wizard/owncloudwizard.cpp

```diff
@@ -337,6 +332,37 @@ void OwncloudWizard::changeEvent(QEvent *e)
 void OwncloudWizard::customizeStyle()
 {
     // HINT: Customize wizard's own style here, if necessary in the future (Dark-/Light-Mode switching)
+    auto theme = Theme::instance();
+    auto wizardPalette = palette();
+
+    wizardPalette.setColor(QPalette::Window, theme->wizardHeaderBackgroundColor());
+    QColor colorWindowDisabled(wizardPalette.color(QPalette::Window));
+    colorWindowDisabled.setAlpha(128);
```

Comment:
I do this to get the text of disabled widgets appear semi transparent. It's necessary for the advanced setup page to let it grey out the check boxes. I have not spotted any problems. Can you explain me what trouble this can cause? What for corresponding methods should I use then instead?

To be honest the whole wizard is asking for trouble :) We do so many things that go against the way Qt expect us to work and there is a lot of dead code and workarounds.

Reply:
@FlexW Do we have any place where we have disabled (greyed-out) text in the UI? Maybe get the color from there. I guess @er-vin wanted to say that you should use a different shade of color instead of using alpha. Like, lowering down the RGB components to a half to make them look much lighter.

Reply:
I've checked that again and I was actually a little bit to fast with my comment. Modifying the disabled color of QPalette::Window is not necessary to let disabled regions appear correct. I removed these few lines but the other modifications to the alpha channel are necessary.  BTW: I use the alpha channel for that because @jancborchardt told me that it looks better and I think too that it looks better than setting some different color for disabled regions.